src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DefaultRazorDynamicFileInfoProvider.cs

```diff
@@ -277,7 +283,7 @@ public Task RemoveDynamicFileInfoAsync(ProjectId projectId, string projectFilePa
 
     public TestAccessor GetTestAccessor() => new(this);
 
-    private void ProjectManager_Changed(object sender, ProjectChangeEventArgs args)
+    private void ProjectManager_Changed(object? sender, ProjectChangeEventArgs args)
```

Comment:
Is this because the event isn't nullable enabled?

Reply:
Pretty sure it's because `EventHandler<T>` is nullable enabled.

Reply:
That's right. Event handlers are always `object? sender` in properly nullable code.